charts/kafka-connect/templates/kafka-connect.yaml

```diff
@@ -1,18 +1,14 @@
 apiVersion: kafka.strimzi.io/v1beta2
 kind: KafkaConnect
 metadata:
-  name: kafka
+  name: {{ .Values.kafkaConnectInstanceName }}
   annotations:
     strimzi.io/use-connector-resources: "true"
 spec:
   version: 3.4.0
   image: "{{ .Values.image.repository }}@{{ .Values.image.digest }}"
   replicas: 1
-  {{- if .Values.bootstrapNamespace }}
-  bootstrapServers: "kafka-kafka-plain-bootstrap.{{ .Values.bootstrapNamespace }}.svc.cluster.local:9092"
-  {{- else }}
   bootstrapServers: "kafka-kafka-plain-bootstrap:9092"
-  {{- end }}
   config:
     group.id: connect-cluster
```

Comment:
Should this be templated to avoid it being combined with the staging kafka connect?

Reply:
this is an internal address within the kafka-cluster. It is the regardless of which namespace it is in. The kafka-cluster and kafka-connect needs to be in the same namespace.

Reply:
I don't think that it matters as the testflight kafka-connect is not reading or writing anything so it does not interfere with the staging cluster.